letsencrypt/cli.py

```diff
@@ -1139,6 +1147,26 @@ def main(cli_args=sys.argv[1:]):
 
     return args.func(args, config, plugins)
 
+def check_config_sanity(args):
+    """Validate command line options and display error message if
+    requirements are not met.
+
+    :param args: Command line options
+    :type args: :class:`argparse.Namespace`
+
+    """
+    # Domain checks
+    if args.domains is not None:
+        # Check if there's a wildcard domain
+        if any(True for d in args.domains if d.startswith("*")):
```

Comment:
`any(d.startswith("*") for d in args.domains)`, same below;

However, this doesn't really check for wildcard domain. `*asd` will be matched, but is not a valid domain at all ;) So you may want to add some comments about that